packages/rich-text/src/RichTextEditor.jsx

```diff
@@ -64,6 +64,7 @@ const styles = {
   }),
   disabled: css({
     background: tokens.gray100,
+    cursor: 'not-allowed',
```

Comment:
This change needs to be done in the `next` branch, otherwise, it won't show up in the web app until later when we merge the `next` branch back into `master`.

Reply:
done:

https://github.com/contentful/field-editors/pull/1128